corehq/motech/generic_inbound/views.py

```diff
@@ -242,13 +242,14 @@ def _log_api_request(api, request, response):
         is_success = False
         status = RequestLog.Status.ERROR
 
-    response_body = response.content.decode('utf-8')
+    # TODO, revisit this
+    # response_body = response.content.decode('utf-8')
     log = RequestLog.objects.create(
         domain=request.domain,
         api=api,
         status=status,
         response_status=response.status_code,
-        error_message=response_body if not is_success else '',
+        # error_message=response_body if not is_success else '',
```

Comment:
At the moment this is a string like `'{"error": "validation error", "errors": ["Invalid request"]}'`, should we just store the value associated with the `error` key when present?  What was the goal of storing this again (we have the full response stored in the processing attempt)?  Guessing we would want to put this as a column in the list view, but the current version doesn't have that (only `status`).

Reply:
I think the intent was as you suggest, to display it in the report but it doesn't seem strictly necessary as long as that data is available in the processing attempt.

Reply:
I mean at this point it's either drop the column from the DB or change what's stored and display it in the report, so it seems like a change is warranted either way.  Got a preference?

Reply:
> Got a preference?

Drop it ☠️